src/main/java/org/isf/opd/gui/OpdBrowser.java

```diff
@@ -381,7 +381,7 @@ private JButton getJDeleteButton() {
 				Opd opd = (Opd) (model.getValueAt(jTable.getSelectedRow(), -1));
 				String dt = '[' + MessageBundle.getMessage("angal.opd.notspecified.msg") + ']';
 				try {
-					DateFormat currentDateFormat = DateFormat.getDateInstance(DateFormat.SHORT, Locale.ITALIAN);
+					DateFormat currentDateFormat = DateFormat.getDateInstance(DateFormat.SHORT, Locale.getDefault());
```

Comment:
Should this be: `new Locale(GeneralData.LANGUAGE)` ?

Reply:
I realized that in https://github.com/informatici/openhospital-gui/blob/develop/src/main/java/org/isf/menu/gui/MainMenu.java#L137

we set the default language in the whole application, so from now on we can use .getDefault() method.

we can open an issue to standardize this behavior if necessary, wdyt?

Reply:
Good to know (and use in the future); thanks

Reply:
Thanks to you @dbmalkovsky !